test/GraphClient/subscribe.test.ts

```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+const sinon = require('sinon');
+import { assert } from 'chai'
+import { Subscription } from 'apollo-client/util/Observable';
+
+import Spy from './../utils/SpyAssert'
+import AssertAsync from './../utils/AssertAsync'
+import GraphClient from './../../src/GraphClient'
+
+describe('GraphClient.subscribe()', () => {
+  let graphClient;
+  let subscriptionQry;
+
+  beforeEach(() => {
+    const subgraphEndPoint = '/subgraph/v1/subgraph1';
+    graphClient = new GraphClient(subgraphEndPoint);
+    subscriptionQry = 'subscription{stakeRequesteds{id}}';
+  });
+
+  it('should work with correct parameters', async () => {
+    const spyGetClient = sinon.replace(
+      graphClient,
+      'getClient',
+      sinon.fake.resolves(true),
+    );
+    const spySubscribe = sinon.replace(
+      graphClient,
+      'subscribe',
+      sinon.fake.resolves(true),
+    );
+    const querySubscriber = await graphClient.subscribe(subscriptionQry);
```

Comment:
in line 27 the `subscribe` function is already mocked. So this test is incorrect.

Reply:
It's taken care 🚀